app/src/main/res/values/strings.xml

```diff
@@ -436,4 +436,5 @@
   <string name="coming_soon">Coming Soon</string>
   <string name="recommended_stories">Recommended Stories</string>
   <string name="stories_for_you">Stories For You</string>
+  <string name="revision_card_activity_title">Skill page</string>
```

Comment:
@rt4914 why 'skill page'? That doesn't seem very informational given the context. Maybe 'skill revision page' would be clearer?

Reply:
@rt4914 shall I rephrase it to skill revision page

Reply:
I agree that `Skill revision page` would give more context as the previous page/tab is related to revision.

Reply:
Done